doc/version.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Could you please add a few words of comment on what kind of version string you are producing

Reply:
See #134